Makefile

```diff
@@ -9,7 +9,7 @@ include $(CURDIR)/make/system-id.mk
 
 # configure some directories that are relative to wherever ROOT_DIR is located
 TOOLS_DIR := $(ROOT_DIR)/tools
-BUILD_DIR := $(ROOT_DIR)/build
+export BUILD_DIR := $(ROOT_DIR)/build
```

Comment:
My preference would be to keep the root BUILD_DIR as non-exported and to export a more specific directory.  That keeps the management of the top of the build dir in this file while allowing sub make files to play under the dirs that are defined by the top level makefile.